bin/webpack-dev-server.js

```diff
@@ -136,6 +138,9 @@ function processOptions(config) {
 function startDevServer(config, options) {
   const log = createLogger(options);
 
+  addEntries(config, options);
+  patchPublicPath(config, options);
```

Comment:
Invalid, no utils will be here

Reply:
So that means that a fix for #1385 and possible fix for webpack-contrib/webpack-hot-middleware#338 is going out on next major release?

Reply:
If not how could I make this not a breaking change? Or is this completely outdated?

Reply:
@arnihermann answered in original issue, it is completely outdated and we should solve this in other way